models/config/binance_parser.py

```diff
@@ -77,13 +77,13 @@ def parser(app, binance_config, args={}):
 
     if 'api_key_file' in binance_config:
         try :
-            with open( binance_config['api_key_file'], 'r') as f :
+            with open(binance_config['api_key_file'], 'r') as f :
```

Comment:
this should not have any influence on the behaviour of the code before and after the change.
please, test again to make sure anything changes because of this?

Reply:
![Screen Shot 2021-07-07 at 7 18 57 PM](https://user-images.githubusercontent.com/8996626/124802489-515b8300-df58-11eb-898d-d66456c9e14c.png)
Hi @lazydroid here is a before and after. Doesn't do that much.  I would assume the spaces wouldn't cause an issue.  I don't appear to have the error in finding the key anymore though
![Screen Shot 2021-07-07 at 5 40 08 PM](https://user-images.githubusercontent.com/8996626/124802681-8667d580-df58-11eb-96a5-1bb8a8b41806.png)